src/arch/src/aarch64/regs.rs

```diff
@@ -455,6 +455,7 @@ mod tests {
     }
 
     #[test]
+    #[should_panic]
```

Comment:
By adding a `should_panic` directive here, I think the test will pass even if other assertions in this test panic, right? Which is not really what we want

Reply:
Indeed, I put the line that should panic in a different test.